packages/portal-proto/src/features/GenomicTables/GenesTable/GTableContainer.tsx

```diff
@@ -111,6 +111,8 @@ export const GTableContainer: React.FC<GTableContainerProps> = ({
       return await createSet({
         filters: buildCohortGqlOperator(genomicFilters),
         case_filters: buildCohortGqlOperator(cohortFilters),
+        intent: "user",
+        set_type: "frozen",
```

Comment:
if i'm reading correctly, I think 'portal' here (https://cdis.slack.com/archives/C01PGJ5KKHD/p1711985112798449?thread_ts=1711729341.125529&cid=C01PGJ5KKHD)

not sure about 'frozen' if it hasn't been saved as part of a cohort yet (but maybe this is edge-casey)

Reply:
Hm, so we want portal as the intent when we create a set for a user when they are creating a cohort? To me creating it as frozen makes sense because the set is specifically created to be put as a filter in a saved cohort.

Reply:
admittedly set operations muddles this.

Reply:
yeah i'm fine with frozen. i was thinking of the situation where the set is created before it's saved to/with the cohort and then something bad happens or we don't end up saving. but i suspect this is not gonna happen, right? :)

i think of 'user' as "we created this set because the user is purposely creating this thing they understand as a 'set'" and 'portal' as "we created this set cos the user needs it but they didn't explicitly ask for it". and grey areas go to "portal".

Reply:
Well, the set creation does happen before the user saves the set right now. So I guess a user could create the set and then decline to save it. I think we should probably just change the set creation to happen when a user saves a set and keep the set_type as frozen.

Reply:
sounds ok to me. and would cut down on..."orphaned" sets.